### ISSUE TITLE
No 15 minutes forecast

### ISSUE THREAD

**Issue by orl0pl** [Neurs12/open-meteo-dart#15]
I can't access `minutely_15` from response of `request` method in `WeatherApi` class.

**Comment by neursh** [comment]
Can you give me steps to reproduce your issue?

**Comment by kemolumi** [comment]
I'm currently working on this feature, it needs some tuning here and there, I'll release it soon.

**Comment by neursh** [comment]
Okay, it should be available in `2.0.2-dev.1`:
```
dependencies:
  open_meteo: ^2.0.2-dev.1
```

Example:
```dart
import 'package:open_meteo/open_meteo.dart';

void main() async {
  final weather = WeatherApi();
  final response = await weather.request(
      latitude: 52.52,
      longitude: 13.41,
      minutely15: {WeatherMinutely15.temperature_2m});
  final data = response.minutely15Data[WeatherMinutely15.temperature_2m]!;
  final minutely15Temperature = data.values;

  print(minutely15Temperature);
}
```

Feel free to reopen this issue if you have any problem while using it!

**Comment by kemolumi** [comment]
Okay, it should be available in `2.0.2-dev.1`:
```
dependencies:
  open_meteo: ^2.0.2-dev.1
```

Example:
```dart
import 'package:open_meteo/open_meteo.dart';

void main() async {
  final weather = WeatherApi();
  final response = await weather.request(
      latitude: 52.52,
      longitude: 13.41,
      minutely15: {WeatherMinutely15.temperature_2m});
  final data = response.minutely15Data[WeatherMinutely15.temperature_2m]!;
  final minutely15Temperature = data.values;

  print(minutely15Temperature);
}
```

Feel free to reopen this issue if you have any problem while using it!
